primes.rb

```diff
@@ -1,6 +1,16 @@
 class Primes
   def self.sum_to(limit = 100)
     # TODO - add your prime number solution here...
-    "I'm working on it!"
+    require 'prime'
+
+	total = 1
+
+	(1..limit).each do |i|
+		break if ((total + i) > limit)  
+		total += i if (Prime.prime?(i))
```

Comment:
This is concise when using the `prime?` method, nice. How would it affect your implementation if you needed to code your own prime? method?